src/runtime/vm/memory_manager.cc

```diff
@@ -105,15 +105,16 @@ NDArray StorageObj::AllocNDArray(size_t offset, std::vector<int64_t> shape, DLDa
   return ret;
 }
 
-MemoryManager* MemoryManager::Global() {
+std::shared_ptr<MemoryManager> MemoryManager::Global() {
```

Comment:
"Global" means global. Thread local object is not obvious behaviour for this method.